### PR TITLE
Specify Initial Sequence Number

### PR DESCRIPTION
**Pull request by sammirata**
With this enhancement, you can now specify the initial sequence number
to be used on the generated segments when calling the packager.
With the old implementation, it was always starting with "1".

### CODE REVIEW

File: packager/app/muxer_flags.cc [hunk 1]
```diff
@@ -50,3 +50,7 @@ DEFINE_bool(mp4_use_decoding_timestamp_in_timeline,
             "be used when generating media timeline, e.g. timestamps in sidx "
             "and mpd. This is to workaround a Chromium bug that decoding "
             "timestamp is used in buffered range, https://crbug.com/398130.");
+DEFINE_int32(mp4_sequence_number_initial,
+             1,
```
Comment (vaage) [inline packager/app/muxer_flags.cc]: Could you rephrase the description to say something like "Sets the initialize sequence number in the mp4 moof->mfhd. Otherwise defaults to 1.".

Comment (vaage) [inline packager/app/muxer_flags.cc]: I find this name a little awkward, could you rename this to "mp4_initial_sequence_number"?

**Comment by kqyang** [comment]
Moved to in #519
